.expeditor/scripts/build_component.sh

```diff
@@ -20,6 +20,7 @@ channel=$(get_release_channel)
 
 echo "--- Channel: $channel - bldr url: $HAB_BLDR_URL"
 
+declare -g hab_binary
```

Comment:
Might be nice to wrap this in a helper function and call _that_ instead, just to help centralize the logic, and make it easier to evolve over time (I'd love it if we could get away from a global variable if possible... but I realize this is `bash`, after all :frowning: )